data/human/human missions.txt

```diff
@@ -5547,3 +5547,130 @@ mission "Quicksilver Mixup 1"
+	source
+		government "Syndicate"
+	to offer
+		random < 10
+	on offer
+		event "argosy hijacking trigger" 14 35
+		fail
+
+event "argosy hijacking trigger"
+
+mission "Argosy Hijacking"
+	name "Stop ship hijacking"
+	description "The <npc> was hijacked on <origin>; disable it and return to <origin>. Do not capture or destroy it; Syndicated Security will deal with the hijackers in space after you have disabled it."
+	landing
+	source
+		government "Syndicate"
+		not attributes "station"
+		near "Alcyone" 4
+		near "Almach" 3
+		near "Gienah" 3
+	to offer
+		has "Argosy Hijacking (Trigger): offered"
+		has "event: argosy hijacking trigger"
+	on offer
+		conversation
+			`You're signing the landing papers with the Syndicated Security officer on <origin>, a standard procedure after landing in a planet within the Syndicate bloc. "Everything seems to be in order. And of course, we'll have your ship refuelled promptly. Have a good day, Captain <last>." He nods his head at you and starts to move off towards the next ship.`
+			`	Just before he is able to, however, you hear a commotion going on by the <npc model> on the landing pad next to you.`
+			choice
+				`	(Turn around to see what's going on.)`
+					goto turn
+				`	(Ignore the commotion.)`
+					goto ignore
+			label turn
+			`	Both of you turn your head to see what's happening, but as you do, there's a deafening blare of warning sirens through <origin>'s spaceport. Suddenly, the <npc model> on the landing pad behind you, the <npc>, jolts up with its loading ramp still extended.`
+				goto hijack
+			
+			label ignore
+			`	Figuring it's nothing, you turn back towards the <ship> only to hear a deafening blare of warning sirens through <origin>'s spaceport. Looking around instinctively, you notice from the corner of your eye that the <npc>, that <npc model> on the landing pad next to you, jolts up with its loading ramp still extended.`
+
+			label hijack
+			`	"Hey, what's going on here?" the officer tries to shout over the noise of the sirens and the repulsors, but the rogue <npc model> veers rapidly and uncertainly towards his direction and the <ship>. He barely jumps down for cover before the ship comes up towards him to clip a stack of cargo crates, sending them flying into the refueling platform. With its repulsors at full power, it narrowly avoids colliding with the <ship> and flees away into the sky.`
+			`	The Syndicated Security officer, dazed and coated in dust kicked up from the unpermitted departure, only just pulls himself up before another person runs up to accost him. "Oi, those kids, they just hijacked my <npc model>! Get someone on their tail!"`
+			choice
+				`	"I'll do it!"`
+				`	(Let Syndicated Security deal with it.)`
+					goto securityjob
+			action
+				set "argosy hijacking accepted"
+			`	The captain turns towards you. "Thank you, Captain! Don't destroy the <npc>, I'm already late for my bloody contract. Agh, this one doesn't help at all," he mutters to himself.`
```

Comment:
What's the second dialogue sentence mean?

Reply:
I'm not sure what you mean.

Reply:
What's he referring to when he says, "this one doesn't help at all"? What's the "one"? Are there more like it?

Reply:
He's already late, so he's been held back some time before for this contract.

I could just remove 'one'.